app/controllers/admin/minor.py

```diff
@@ -4,33 +4,35 @@
 
 from app.controllers.admin import admin_bp
 
-from app.logic.minor import getMinorInterest, getMinorProgress, toggleMinorInterest
+from app.logic.minor import getMinorInterest, getMinorProgress, toggleMinorInterest, getDeclaredMinorStudents
 
-@admin_bp.route('/admin/cceMinor', methods=['POST','GET'])
+@admin_bp.route('/admin/cceMinor', methods=['GET','POST'])
 def manageMinor():
-
     if not g.current_user.isAdmin:
         abort(403)
     
     if request.method == 'POST':
         interested_students = request.form.getlist('interestedStudents[]')
```

Comment:
I know you didn't really work with this, but would love to not have camelcase here. Also the `i in interested_students:` could use a new variable name in place of `i` too.